app/scripts/directives/nav.js

```diff
@@ -1,25 +1,40 @@
 'use strict';
 
 angular.module('openshiftConsole')
-  .directive('sidebar', function(HawtioNav) {
+  .directive('sidebar', function($location, Constants) {
+    var itemMatchesPath = function(item, path) {
+      return (item.href === path) || _.some(item.prefixes, function(prefix) {
```

Comment:
I was wondering if we should try to normalize URLs as well with `URI.normalizePath`, but probably not worth it. We had some performance problems before doing URL manipulation to highlight nav items.